scripts/github_api

```diff
@@ -139,6 +190,26 @@ if __name__ == '__main__':
                 (('--body',), dict(help='pull request body text')),
             ],
         },
+        'query_pr': {
+            'description': "Get information about a GitHub pull-request",
+            'function': query_pull_request,
+            'arguments': [
+                (('repo',), dict(
+                    help='target repository path: <repo owner>/<repo>')),
```

Comment:
This is a little confusing. I assume the text after the colon (:) is the path format. Pls mention it explicitly. eg "target repository path. Format: <repo owner>/<repo"